src/content/docs/mobile-monitoring/new-relic-mobile-ios/get-started/new-relic-ios-and-dt.mdx

```diff
@@ -12,10 +13,7 @@ Watch this short video (approx. 2:15 minutes) to learn how to:
 
 ## Requirements
 
-Here's what you need to use distributed tracing with New Relic mobile:
-
-    * Android agent version 6.0.0 or higher
-    * XCFramework agent version 7.3.0
+We recommend the most current agent, but to use distributed tracing, you need at least XCFramework agent version 7.3.0.
```

Comment:
This sentence is odd. I'd probably remove the first part (we recommend the most current agent). Or break the sentence in two.

Reply:
Thanks @urbiz-nr !